clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -248,6 +250,13 @@ public synchronized long lastSuccessfulUpdate() {
     }
 
     /**
+     * The max allowable age of metadata.
+     */
+    public synchronized long getMetadataMaxAge() { 
```

Comment:
I am fine either way. 
Just that the getter name can be set to something that is self explainatory.